src/test/meson.build

```diff
@@ -608,14 +608,14 @@ tests += [
           'src/test/nss-test-util.h'],
          [],
          [libdl],
-         [], 'ENABLE_NSS', 'manual'],
+         [], 'ENABLE_NSS'],
```

Comment:
@yuwata I'd wait for @keszybz to weigh in. I'm still not sure why those tests were "manual".

Reply:
Looking at https://the-real-systemd.semaphoreci.com/jobs/f966d40a-b3c7-4414-a741-684d7df60e1b where the tests failed with
```
 FAIL: test-nss-hosts18:06
======== myhostname ========18:06
Failed to load module myhostname: libnss_myhostname.so.2: cannot open shared object file: No such file or directory
======== mymachines ========19:06
Failed to load module mymachines: libnss_mymachines.so.2: cannot open shared object file: No such file or directory
```
It seems they need polishing

Reply:
Probably test suite should set `LD_LIBRARY_PATH` variable pointing to directory with locally built `libnss_myhostname.so.2` and `libnss_mymachines.so.2`.